spec/controllers/errors_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "spec_helper"
+
+describe ErrorsController do
+  describe "404" do
+    it "returns an HTML 404 page for .html" do
+      get :"404", params: { format: :html }
+      expect(response.status).to eq(404)
+      expect(response.header["Content-Type"]).to eq("text/html; charset=utf-8")
+    end
+
+    it "returns an HTML 404 page for .htm" do
+      get :"404", params: { format: :htm }
+      expect(response.status).to eq(404)
+      expect(response.header["Content-Type"]).to eq("text/html; charset=utf-8")
+    end
+
+    it "returns an HTML 404 page for .epub" do
+      get :"404", params: { format: :epub }
+      expect(response.status).to eq(404)
+      expect(response.header["Content-Type"]).to eq("text/html; charset=utf-8")
+    end
+  end
+
+  describe "500" do
+    it "returns an HTML 500 page" do
+      get :"500"
+      expect(response.status).to eq(500)
+      expect(response.header["Content-Type"]).to eq("text/html; charset=utf-8")
+    end
+  end
+
+  describe "auth_error" do
+    render_views
```

Comment:
We generally avoid using `render_views` in controller specs. If we do need to examine the response page, we would either use request specs (spec/requests/) or Cucumber tests (features/). For this test, I think it's fine to skip `render_views` and the page title check - or even skip it entirely (until we change it more, e.g. return status code 401).

Reply:
> even skip it entirely (until we change it more, e.g. return status code 401).

I'll take this advice and remove additions related to "auth_error".